File: server/db/waterCouncilEpisodes.js
const episodes = new Map();

/**
 * @param {object} episode
 * @returns {object}
 */
function createEpisode(episode) {
  episodes.set(episode.id, episode);
  return episode;
}

/**
 * @param {string} id
 * @returns {object | undefined}
 */
function getEpisode(id) {
  return episodes.get(id);
}

/**
 * @returns {object[]}
 */
function listEpisodes() {
  return Array.from(episodes.values());
}

module.exports = {
  createEpisode,
  getEpisode,
  listEpisodes,
};

File: server/db/water_council_episodes.py
episodes: dict = {}


def create_episode(episode: dict) -> dict:
    episodes[episode["id"]] = episode
    return episode


def get_episode(episode_id: str) -> dict | None:
    return episodes.get(episode_id)


def list_episodes() -> list[dict]:
    return list(episodes.values())
